Resources/Prototypes/Entities/Mobs/NPCs/xenopet.yml

```diff
@@ -283,61 +347,62 @@
 #      path: /Audio/Animals/lizard_happy.ogg
-#
-#- type: entity
-#  name: FXE Subject 7355
-#  parent: MobXenoNeutralRavager
-#  id: MobXenoSubjectTess
-#  description: An extremely oddly coloured xeno, with glowing stripes. An odd mutation.
-#  components:
-#  - type: NpcFactionMember
-#    factions:
-#     - PetsNT
-#  - type: Tool
-#    speed: 3
-#    qualities:
-#      - Prying
-#  - type: Sprite
-#    drawdepth: Mobs
-#    sprite: Mobs/Animals/subject7355.rsi
-#    layers:
-#    - map: ["enum.DamageStateVisualLayers.Base"]
-#      state: subject7355
-#    - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
-#      state: glow
-#      shader: unshaded
-#  - type: PointLight
-#    radius: 2
-#    energy: 1
-#    color: "#639fff"
-#  - type: DamageStateVisuals
-#    states:
-#      Alive:
-#        Base: subject7355
-#      Critical:
-#        Base: subject7355_crit
-#      Dead:
-#        Base: subject7355_dead
-#  - type: Inventory
-#    templateId: friendxeno
-#  - type: IdExaminable
-#  - type: InventorySlots
-#  - type: Strippable
-#  - type: UserInterface
-#    interfaces:
-#    - key: enum.StrippingUiKey.Key
-#      type: StrippableBoundUserInterface
-#  - type: Grammar
-#    attributes:
-#      proper: true
-#      gender: male
-#  - type: InteractionPopup
-#    successChance: 0.5
-#    interactSuccessString: petting-success-reptile
-#    interactFailureString: petting-failure-generic
-#    interactSuccessSpawn: EffectHearts
-#    interactSuccessSound:
-#      path: /Audio/Animals/lizard_happy.ogg
+
+- type: entity
+  name: FXE Subject 7355
+  parent: MobXenoNeutralRavager
+  id: MobXenoSubjectTess
+  description: An extremely oddly coloured xeno, with glowing stripes. An odd mutation.
+  components:
+  - type: NpcFactionMember
+    factions:
+     - PetsNT
+  - type: Tool
+    speed: 3
+    qualities:
+      - Prying
+  - type: Sprite
+    drawdepth: Mobs
+    sprite: Mobs/Animals/subject7355.rsi
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: subject7355
+    - map: [ "enum.DamageStateVisualLayers.BaseUnshaded" ]
+      state: glow
+      shader: unshaded
+  - type: PointLight
+    radius: 2
+    energy: 1
+    color: "#639fff"
+  - type: DamageStateVisuals
+    states:
+      Alive:
+        Base: subject7355
+      Critical:
+        Base: subject7355_crit
+      Dead:
+        Base: subject7355_dead
+  - type: Inventory
+    templateId: friendxeno
+  - type: IdExaminable
+  - type: InventorySlots
+  - type: Strippable
+  - type: UserInterface
+    interfaces:
+    - key: enum.StrippingUiKey.Key
+      type: StrippableBoundUserInterface
+  - type: Grammar
+    attributes:
+      proper: true
+      gender: male
+  - type: InteractionPopup
+    successChance: 0.5
+    interactSuccessString: petting-success-reptile
+    interactFailureString: petting-failure-generic
+    interactSuccessSpawn: EffectHearts
+    interactSuccessSound:
+      path: /Audio/Animals/lizard_happy.ogg
+
 #
 #- type: entity
```

Comment:
Please make sure to move all of these events out of these files, at least when you backport this to EE. 

All events must be placed in events.yml.